### PR TITLE
Create Currency.cpp

### PR DESCRIPTION
**Pull request by AhmedHamdyKotp**

### CODE REVIEW

File: src/Currency.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+void Currency::setAmount(long amt) {
+    checkForOverflow(amt);
+    amount = amt;
+}
+
+// Initialize exchange rates from CSV
+void Currency::initializeExchangeRates() {
+    ifstream file("output.csv");
```
Comment (AhmedHamdyKotp) [inline src/Currency.cpp]: Here should be modified by the csv file location on the repo

File: src/Currency.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    currencyCode = code;
+}
+
+void Currency::setAmount(long amt) {
+    checkForOverflow(amt);
+    amount = amt;
+}
+
+// Initialize exchange rates from CSV
+void Currency::initializeExchangeRates() {
```
Comment (AhmedHamdyKotp) [inline src/Currency.cpp:80]: .

**Comment by OsamaAshraf01** [comment]
Modify it 😄

**Comment by OsamaAshraf01** [comment]
Download the project on your pc and work on it

File: src/Currency.cpp (hunk 1 above)
Comment (OsamaAshraf01) [reply src/Currency.cpp]: Which file ?

**Review by AhmedHamdyKotp — COMMENTED** [review]
i modified it bro

**Pull request merged**